CHANGELOG.md

```diff
@@ -28,6 +28,9 @@ guest properties [#235](https://github.com/vmware/go-vcloud-director/pull/235)
 * Added methods `vapp.SetProductSectionList` and `vapp.GetProductSectionList` allowing to manipulate
 vApp guest properties [#235](https://github.com/vmware/go-vcloud-director/pull/235)
 * Added method GetStorageProfileByHref
+* Added methods `CreateNsxvNatRule()`, `UpdateNsxvNatRule()`, `GetNsxvNatRuleById()`, `DeleteNsxvNatRuleById()`
```

Comment:
Maybe comment should state that this for adcanced edge gtw

Reply:
Added